resources/assets/coffee/react/beatmap-discussions/discussion.coffee

```diff
@@ -167,7 +168,7 @@ export class Discussion extends React.PureComponent
 
     topClasses = "#{vbn} #{vbn}--#{type}"
     topClasses += " #{vbn}--inactive" if score != 0
-    user = @props.users[@props.discussion.user_id]
+    user = @props.users[@props.discussion.user_id] ? @props.users[null]
```

Comment:
at least around line 290 of this file also need this

https://github.com/ppy/osu-web/blob/f4dd0e1a74635ea61acb6e2263ecfa7264a6beae/resources/assets/coffee/react/beatmap-discussions/discussion.coffee#L290-L291

Then maybe here (and there's random `currentUser.id?` boolean comparison against number)

https://github.com/ppy/osu-web/blob/f4dd0e1a74635ea61acb6e2263ecfa7264a6beae/resources/assets/coffee/react/modding-profile/posts.coffee#L55-L56